docker-unified/Dockerfile

```diff
@@ -0,0 +1,263 @@
+###############################################
+#                 Build stage                 #
+###############################################
+FROM node:16-slim AS node-build
+
+# TODO: Change default branch name before merge into master
+ARG web_branch=master
```

Comment:
@vincentsalucci Do we need to do something different here?

Reply:
I have updated the Dockerfile to pull the latest Web artifact from GitHub.